static/js/zamboni/devhub.js

```diff
@@ -990,7 +990,9 @@ function initCatFields(delegate) {
             $("input", $misc).prop("checked", checkedLength <= 0);
         };
         var checkOther = function() {
-            $("input", $main).prop("checked", false).prop("disabled", false);
+            // Get main categories
+            var $mainCat = $main.find('input[name="form-0-categories"]');
+            $($mainCat).prop("checked", false).prop("disabled", false);
```

Comment:
`$mainCat` already is a jquery object so you don't need to wrap it a second time afaict